ansible/roles/prosody/files/mod_muc_auth_vpaas.lua

```diff
@@ -45,6 +49,23 @@ end
 
 timer.add_task(CACHE_EXPIRATION_SECONDS, invalidate_cache)
 
+-- gets or creates an entry in the cache for the specified kid
+-- checks is it vpass and extract the tenant (everything before the /) in the kid
+local function get_kid_kid_parse_cache_obj(kid) then
+    local kid_parse_cache_obj = kid_parse_cache:get(kid);
+    if not kid_parse_cache_obj then
+        kid_parse_cache_obj = {};
+
+        kid_parse_cache_obj.is_vpaas = starts_with(kid, VPAAS_PREFIX);
+
+        local tenant, _ = kid:match("^(.*)%/(.*)$")
```

Comment:
Why define the second group if you don't need it?
Should be the same as `local tenant = kid:match("^(.*)%/.*$")`